demos/LowAllocationWebServer.Tests/GivenAnHttpHeaders.cs

```diff
@@ -21,48 +20,47 @@ public class GivenAnHttpHeaders
 Accept-Language: en-US,en;q=0.8,pt-BR;q=0.6,pt;q=0.4
 ";
 
-        private string _headerWithoutColumn = "Connection keep-alive\r\n";
+        private const string HeaderWithoutColumn = "Connection keep-alive\r\n";
 
-        private string _headerWithoutCRLF = "Host: localhost:8080";
+        private const string HeaderWithoutCrlf = "Host: localhost:8080";
 
-        private ByteSpan _headers;
         private HttpHeaders _httpHeaders;
 
-        [TestInitialize]
-        public void Setup()
-        {                        
-            var bytes = new UTF8Encoding().GetBytes(_headersString);            
+        public GivenAnHttpHeaders()
+        {
+            ByteSpan headers;
+            var bytes = new UTF8Encoding().GetBytes(HeadersString);            
 
             unsafe
             {
                 fixed (byte* buffer = bytes)
                 {
-                    _headers = new ByteSpan(buffer, bytes.Length);
+                    headers = new ByteSpan(buffer, bytes.Length);
                 }
             }
 
-            _httpHeaders = new HttpHeaders(_headers);
+            _httpHeaders = new HttpHeaders(headers);
```

Comment:
I will log an issue and add a TODO to switch to Slices.Span<T> once it is integrated in the Formatter APIs.